.github/CODEOWNERS

```diff
@@ -4,5 +4,6 @@
 # For syntax help see:
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
+.github/auto-approve.yml  @googleapis/github-automation
```

Comment:
I can't find this one, should it be `@yoshi-automation` ?

Reply:
I found it! It is a team, not a user. What role does it need to have?

Reply:
The team needs explicit write access

Reply:
Done!